std_msgs/Header header
RangeMeasurement[] measurements
